maquetado/scss/books.scss

```
@@ -1,10 +1,12 @@
 .book-container {
+  display: grid;
   background-color: $white;
   box-shadow: 4px 4px 10px 0 $black-05;
-  display: flex;
+  grid-template-columns: 0.4fr 0.6fr;
```

Perfect! You can also make an approach that sometimes gives more semantic meaning to CSS grids 

```suggestion
  grid-template-areas:
    'cover cover info info info';
```

And then set `grid-area` in its inner elements, but this is fine too. Let's leave it is.